components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/introspection/OAuth2IntrospectionEndpoint.java

```diff
@@ -143,10 +145,35 @@ public Response introspect(@FormParam("token") String token, @FormParam("token_t
                 .setIssuedAt(introspectionResponse.getIat())
                 .setExpiration(introspectionResponse.getExp())
                 .setAuthorizedUserType(introspectionResponse.getAut())
-                .setAudience(introspectionResponse.getAud());;
+                .setAudience(introspectionResponse.getAud());
 
         if (introspectionResponse.getAuthorizedUser() != null) {
-            respBuilder.setOrgId(introspectionResponse.getAuthorizedUser().getAccessingOrganization());
+            String accessingOrganizationId = introspectionResponse.getAuthorizedUser().getAccessingOrganization();
```

Comment:
This logic needs to be handled when building the introspection response. Otherwise services which get the introspection response internally will not receive these details.